lib/valueflow.cpp

```diff
@@ -89,14 +89,35 @@ static void execute(const Token *expr,
                     MathLib::bigint *result,
                     bool *error);
 
-static void bailout(TokenList *tokenlist, ErrorLogger *errorLogger, const Token *tok, const std::string &what)
+static void bailoutInternal(TokenList *tokenlist, ErrorLogger *errorLogger, const Token *tok, const std::string &what, const char *file, size_t line, const char *function)
```

Comment:
My default was to use the tightest type (and I have no use for negative values here).
So everybody can see it should be a nonnegative quantity.
_But_ since "__LINE__" here comes from the preprocessor, I agree.

Ok, will change the type of "function" to std::string.